Snowflake.Data/Core/SFBindUploader.cs

```diff
@@ -224,13 +224,13 @@ internal async Task UploadStreamAsync(MemoryStream stream, string destFileName,
             statement.SetUploadStream(stream, destFileName, stagePath);
             await statement.ExecuteTransferAsync(putStmt, cancellationToken).ConfigureAwait(false);
         }
-        private string GetCSVData(string sType, string sValue)
+        
+        internal string GetCSVData(string sType, string sValue)
         {
             if (sValue == null)
                 return sValue;
 
-            DateTime dateTime = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Unspecified);
-            DateTimeOffset dateTimeOffset = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Unspecified);
+            DateTime epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Unspecified);
```

Comment:
Can we keep the epoch as a const somewhere, also wouldn't it be more predictable to keep it as `DateTimeKind.Utc`? In .Net standard 2.1, the `Utc` kind is used in `DateTime.UnixEpoch`.